src/harness/harnessIO.ts

```diff
@@ -596,7 +596,7 @@ export namespace Compiler {
                 .map(s => "!!! " + ts.diagnosticCategoryName(error) + " TS" + error.code + ": " + s);
             if (error.relatedInformation) {
                 for (const info of error.relatedInformation) {
-                    errLines.push(`!!! related TS${info.code}${info.file ? " " + ts.formatLocation(info.file, info.start!, formatDiagnsoticHost, ts.identity) : ""}: ${ts.flattenDiagnosticMessageText(info.messageText, IO.newLine())}`);
+                    errLines.push(`!!! related TS${info.code}${info.file ? " " + ts.formatLocation(info.file, info.start!, formatDiagnsoticHost, ts.identity, isDefaultLibraryFile(info.file.fileName)) : ""}: ${ts.flattenDiagnosticMessageText(info.messageText, IO.newLine())}`);

```

Comment:
cant you instead do string replacement on all lines for errs to replace  `/.ts/lib.*.d.ts positions`

Reply:
I can, it just seemed like it'd be a lot slower. The typeWriter already uses `/lib(.*)\.d\.ts/i` to detect this, but I was just unsure whether or not that was advisable if a test defined its own `lib.d.ts` files that weren't actually in `/.ts`.

Maybe I can just do:

```ts
if (isDefaultLibraryFile(info.file)) {
	location = location.replace(/(lib(?:.*)\.d\.ts):\d+:\d+:/i, "$1:")
}
```

If that's preferrable?

Reply:
Yeah .. would prefer not having to add this to `program.ts`

Reply:
Done.